index.html

```diff
@@ -869,7 +893,24 @@ <h3>Script Event</h3>
 
         <section>
           <h4>Text</h4>
-         <p>The <dfn>Text</dfn> object contains text content typically in a single language. This language may be the <a>Original</a> language or a <a>Translation</a> language.</p>
+          <p>The <dfn>Text</dfn> object contains text content typically in a single language.
+            This language may be the <a>Original</a> language or a <a>Translation</a> language.</p>
+          <p><a>Text</a> is defined as <dfn>Original</dfn> if it is any of:
+            <ol>
+              <li>the same language as the dialogue that it represents in the original programme audio;</li>
+              <li>a transcription of text visible in the programme video, in the same language as that text;</li>
+              <li>an untranslated representative of non-dialogue sound;</li>
+              <li>an untranslated description of the scene in the programme video.</li>
+            </ol>
+          </p>
+          <aside class="note">The language of an <a>Original</a> <a>Text</a> object
+            can be different to the document's <a>Default Language</a>.</aside>
```

Comment:
Although this _defines_ Original and Translation, it doesn't say how any given Text object can be _identified_ as either Original or Translation.

Reply:
Addressed in d58c992